src/components/shared/pagination/Pagination.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import Pagination from './PaginationComp';
+
+type props = {
+  itemsPerPage: number;
+};
+export default function PaginatedData({ itemsPerPage }:props) {
+  const [data] = useState([
+    'Item 1',
+    'Item 2',
+    'Item 3',
+    'Item 4',
+    'Item 5',
+    'Item 6',
+    'Item 7',
+    'Item 8',
+    'Item 9',
+    'Item 10',
+    'Item 11',
+    'Item 12',
+    'Item 13',
+    'Item 14',
+    'Item 15',
+  ]);
+
```

Comment:
@emilenfc nice work but I think you can create does item data in different file; and them import them from this file, 
for more clean and reliable code. 
Thank you.

Reply:
@ISHIMWE-Aime, let me do it....

Reply:
@ISHIMWE-Aime, let me do it